src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -40,7 +40,7 @@
 
 register_cli_argument('resource list', 'name', resource_name_type)
 register_cli_argument('resource move', 'ids', nargs='+')
-register_cli_argument('resource invoke-action', 'action', help='The action that will be invoked')
+register_cli_argument('resource invoke-action', 'action', help='The action that will be invoked on the specified resource')
```

Comment:
That doesn't really make it much more clear. How will someone know what the possible actions are for a resource?  Is there another endpoint that documents this?

Reply:
This is a generic command as a stop gap where there is no strong type CLI command for that. So it targets to advance users or who are suggested by such people. Under that context, people are expected to check out the rest api themselves which are well documented in msdn, like
https://docs.microsoft.com/en-us/rest/api/compute/virtualmachines/virtualmachines-start.
We should call out such sources in the help though.